cape-ph/cape-cod
# 126 refactor alb static app deployment acm cert stuff etc out to reusable place

# TO TEST

This was a large refactor just to make a lot of non-reusable stuff reusable. There should be no functional change, but a number of resources did change due to namespacing, new classes, logic getting relocated, etc.

This is currently deployed, so after reviewing code, polkaround
